src/utils/feedbackFormURL.ts

```diff
@@ -0,0 +1,16 @@
+export const feedbackFormURL = (
+  referrer: 'deploy' | 'stoned-banny',
+  projectHandle?: string,
+  userAddress?: string,
+) => {
+  let url = `https://auditor.typeform.com/to/REMUTIbQ#`
```

Comment:
**suggestion**: it might be helpful to use the `URLSearchParams` api for building the query string:

https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams/URLSearchParams#examples

Reply:
Nice suggestion. Implemented